.cirrus.yml

```diff
@@ -12,7 +12,7 @@ tool_setup_template: &TOOL_SETUP_TEMPLATE
   tool_setup_script:
     - git fetch origin master # To set FETCH_HEAD for "git merge-base" to work
     - cd script/tool
-    - pub get
+    - dart pub get
```

Comment:
Thanks :) I've been slowly fixing this when I remember, but that's not often enough.

Reply:
Yeah, I should really keep these PRs more scoped to the main task, but it's easy to just do some small cleanup as I'm working on something else.